infra/proposer.go

```diff
@@ -17,13 +18,28 @@ type Proposers struct {
 	logger *log.Logger
 }
 
-func CreateProposers(conn, client int, nodes []Node, crypto *Crypto, logger *log.Logger) *Proposers {
+func GetTLSCACerts(files string) ([][]byte, error) {
+	var certs [][]byte
+	in, err := ioutil.ReadFile(files)
+	if err != nil {
+		return nil, err
+	}
+
+	certs = append(certs, in)
+	return certs, nil
+}
```

Comment:
this is probably a nit but the code style could be better:
it is strange to consume **one** file and return **multiple** certs. We need 2D array **only** because a consumer (`grpc client`) far away requires so, and this should be done there. We should try to limit the scope of function, and minimize side effects.

Reply:
currently it will return a byte array as file IO. the 2D changes in `CreateGRPCClient`